.github/workflows/check.yml

```diff
@@ -70,8 +70,6 @@ jobs:
       - name: Test
         run: pnpm vitest --shard ${{ matrix.shard }}
         if: matrix.runtime == 'Node'
-        env:
-          NODE_OPTIONS: --max_old_space_size=8192
```

Comment:
This had no effect but also the random test fails seem to be appearing way less recently.